mlflow/pyfunc/utils/data_validation.py

```
@@ -88,17 +88,19 @@ def wrapper(self, *args, **kwargs):
             try:
                 model_validate(ChatAgentRequest, args[0])
                 request = ChatAgentRequest(**args[0])
+            except pydantic.ValidationError as e:
```

What's the difference of this change?

---

if there was a pyfunc validation error within the actual predict/predict_stream call, then we'd unintentionally swallow that error and preface it with the "invalid dictionary input for a .." message

---

Make sense